vector/src/main/java/im/vector/app/features/home/room/list/RoomListFragment.kt

```diff
@@ -473,9 +473,16 @@ class RoomListFragment @Inject constructor(
             // refresh footer
             footerController.setData(it)
         }
-        RoomListQuickActionsBottomSheet
+        val bottomSheet = RoomListQuickActionsBottomSheet
                 .newInstance(room.roomId, RoomListActionsArgs.Mode.FULL)
-                .show(childFragmentManager, "ROOM_LIST_QUICK_ACTIONS")
+        bottomSheet.listener = object : RoomListQuickActionsBottomSheet.Listener {
+            override fun handleFailure(throwable: Throwable) {
+                showFailure(throwable)
```

Comment:
Nothing happen when trying to change the notification setting from the bottom sheet. From the notification fragment a dialog is displayed in this case.

Reply:
I was not clear sorry.
- enter airplane mode on the device
- change the notification setting from the bottom sheet: nothing happen (the UI is updated though, but it's wrong)
- if you change the notification setting from the full screen settings, a dialog with the network error is displayed.

Reply:
Hrmm I obviously didn't test that properly. I've added a change that fixes the snackbar on the `RoomList` fragment. That actually displays behind the bottom sheet so, just added a dialog in the bottom sheet instead.